lib/screens/home_screen.dart

```diff
@@ -9,6 +9,9 @@ import 'package:taskly/service/random_tip_service.dart';
 import 'package:taskly/widgets/theme_mode_switch.dart';
 import 'package:taskly/widgets/tip_of_day_card.dart';
 
+// Enum to represent options in the menu
+enum TaskOption { deleteAll }
```

Comment:
Could you please make an Enums folder and write this under it? Thanks

Reply:
Done